docs/marketplace/dev/front-end/web-components.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Web Components
+description: This articles provides details how web components are used and what their purpose is in the Merchant Portal Frontend.
+template: concept-topic-template
+---
+
+This articles provides details how web components are used and what their purpose is in the Merchant Portal Frontend.
+
+## Introduction
+
+The main development of Merchant Portal is done withing Angular Components and Angular infrastructure. When a component is needed to be accessed from Twig it is exposed as a Web Component.
+Some of the internal component will always remain as Angular Component unless they will be required to be used from the Twig side.
+
+## Web Components
+
+Web Components is a suite of different technologies allowing you to create reusable custom elements — with their functionality encapsulated away from the rest of your code — and utilize them in your web apps.
+
+The full cycle of the new module creation and registration it as a web component you can see in the [following document](/docs/marketplace/dev/front-end/how-to-create-a-new-module-with-application.html).
```

Comment:
Please provide here a short snippet example how to register a component as a webcomponent.
Also please mention that all web components will get a prefix to their selectors by default and show how to customise it as an example.